flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/adaptive/CreatingExecutionGraphTest.java

```diff
@@ -135,10 +139,16 @@ public void testNotPossibleSlotAssignmentTransitionsToWaitingForResources() thro
     public void testSuccessfulSlotAssignmentTransitionsToExecuting() throws Exception {
         try (MockCreatingExecutionGraphContext context = new MockCreatingExecutionGraphContext()) {
             final CompletableFuture<CreatingExecutionGraph.ExecutionGraphWithVertexParallelism>
-                    executionGraphWithvertexParallelismFuture = new CompletableFuture<>();
-            final CreatingExecutionGraph creatingExecutionGraph =
-                    new CreatingExecutionGraph(
-                            context, executionGraphWithvertexParallelismFuture, log);
+                    executionGraphWithVertexParallelismFuture = new CompletableFuture<>();
+            final AtomicReference<Consumer<Throwable>> errorHandlerRef = new AtomicReference<>();
+            new CreatingExecutionGraph(context, executionGraphWithVertexParallelismFuture, log) {
+                @Override
+                OperatorCoordinatorHandler createOperatorCoordinatorHandler(
```

Comment:
I'd rather add a OperatorCoordinatorHandler factory argument to the CreatingExecutionGraph than resorting to this.

Reply:
👍